src/SFML/Graphics/Sprite.cpp

```diff
@@ -69,7 +69,7 @@ void Sprite::setTexture(const Texture& texture, bool resetRect)
     if (resetRect || (!m_texture && (m_textureRect == sf::IntRect())))
     {
         Vector2i size = Vector2i(texture.getSize());
-        setTextureRect(IntRect(0, 0, size.x, size.y));
+        setTextureRect(IntRect({0, 0}, {size.x, size.y}));
```

Comment:
```suggestion
        setTextureRect(IntRect({0, 0}, Vector2i(texture.getSize())));
```
and removing the line before, now useless.